nvflare/private/fed/client/client_executor.py

```diff
@@ -347,7 +327,7 @@ def abort_app(self, job_id):
                         optional=True,
                     )
                     self.logger.debug("abort sent to worker")
-                    t = threading.Thread(target=self._terminate_process, args=[child_process, job_id])
+                    t = threading.Thread(target=self._terminate_process, args=[job_handle, job_id])
```

Comment:
Why do we need a separate thread here? Can we simply call self._terminate_process()?

Reply:
The abort_app() which calls the self._terminate_process() needs to return to the user response right away. The self._terminate_process() has a potential waiting timeout. Place in a thread to avoid the response delay.